pkg/registry/apis/provisioning/jobs/persistentstore.go

```diff
@@ -20,15 +16,20 @@ import (
 	"k8s.io/apimachinery/pkg/selection"
 	"k8s.io/apiserver/pkg/endpoints/request"
 	"k8s.io/apiserver/pkg/registry/rest"
+
+	"github.com/grafana/grafana-app-sdk/logging"
+	"github.com/grafana/grafana/pkg/apimachinery/identity"
+	provisioning "github.com/grafana/grafana/pkg/apis/provisioning/v0alpha1"
+	"github.com/grafana/grafana/pkg/registry/apis/provisioning/apifmt"
 )
 
 const (
 	// LabelJobClaim includes the timestamp when the job was claimed.
 	// The label must be formatted as milliseconds from Epoch. This grants a natural ordering, allowing for less-than operators in label selectors.
 	// The natural ordering would be broken if the number rolls over into 1 more digit. This won't happen before Nov, 2286.
 	LabelJobClaim = "provisioning.grafana.app/claim"
-	// LabelJobOriginalName contains the Job's name as a label. This allows for label selectors to find the archived version of a job.
-	LabelJobOriginalName = "provisioning.grafana.app/original-name"
+	// LabelJobOriginalUID contains the Job's original uid as a label. This allows for label selectors to find the archived version of a job.
```

Comment:
why this change?

Reply:
LabelJobOriginalUID is a unique identifier from the original job -- it links the specific active job to a specific historic job.

I don't really understand what LabelJobOriginalName is useful for -- it is always a substring of the job name and will have multiple matches (eg every migratesync job).  The previous usage in tests *may* be a bug -- but likely not based on the fact we wipe the DB with each run